paasta_tools/utils.py

```diff
@@ -3345,6 +3348,13 @@ def get_git_sha_for_deploy_group(self, deploy_group: str) -> str:
             e = f"{self.service} not deployed to {deploy_group}. Has mark-for-deployment been run?"
             raise NoDeploymentsAvailable(e)
 
+    def get_image_version_for_deploy_group(self, deploy_group: str) -> str:
+        try:
+            return self.config_dict["deployments"][deploy_group]["image_version"]
+        except KeyError:
+            e = f"{self.service} not deployed to {deploy_group}. Has mark-for-deployment been run?"
+            raise NoDeploymentsAvailable(e)
```

Comment:
Currently none of our config_dict['deployments'] have any `image_version` key, so we will always get this `KeyError`, but not because the `deploy_group` is missing, but because the `image_version` is missing. 

I think we might want to at least split this to two lines and `get('image_version')` so the KeyError + error message apply to the right conditions.

Reply:
I've added a line to check that `image_version` is there, and otherwise return `None`. I think once these changes have propagated and `image_version` gets added to the deployments.json files, then we can remove that check (if I've understood it right). Is that okay?

Reply:
Yup, once this runs once everywhere we should be able to remove it. This works fine, though I still think it is a strange edge case that would give the user the wrong error message if if deployments.json (or whatever alternate source of deployment info we're using) somehow lost keys we think 'always' exist.